Explorer/Assets/Scripts/ECS/Abstract/BaseUnityLoopSystem.cs

```diff
@@ -66,6 +66,24 @@ public sealed override void Update(in float t)
             }
         }
 
+        public sealed override void Dispose()
+        {
+            if (isDisposed)
+            {
+                ReportHub.LogError(ReportCategory.ENGINE, $"Attempt to dispose already disposed system: {this.GetType().Name}");
```

Comment:
By Microsoft's documentation, Dispose is supposed to be idempotent. Calling it multiple times is not meant to be a mistake. We can of course decide otherwise.